### PR TITLE
Update play-dgraph-io.nginx.conf

### PR DESCRIPTION
**Pull request by MichelDiz**
Note: The Dg-Auth token is a user token. With no Admin rights.

### CODE REVIEW

File: server/play-dgraph-io.nginx.conf [hunk 1]
```diff
@@ -1,5 +1,5 @@
 proxy_cache_path /var/nginx levels=1:2 keys_zone=dgraphcache:20m max_size=1g inactive=60m use_temp_path=off;
-limit_req_zone $binary_remote_addr zone=play:10m rate=1000r/s;
+limit_req_zone $binary_remote_addr zone=play:10m rate=800r/s;
```
Comment (skrdgraph) [inline server/play-dgraph-io.nginx.conf:2]: any reason to change from `1000` to `800`? has there been any issue related to this? an additional `200` doesn't seem to be an issue imo.

**Comment by skrdgraph** [comment]
@MichelDiz some things I am not clear about (they are indirectly related to your PR)

- https://github.com/dgraph-io/dgraph uses the `secrets`, I think we should explore that here for the tokens
- Also, what's the deployment mechanism for this config change? Does it get auto-deployed? (if not, we should try to focus on that next)

I don't see any harm in merging this PR.

**Review by skrdgraph — APPROVED** [review]

File: server/play-dgraph-io.nginx.conf (hunk 1 above)
Comment (MichelDiz) [reply server/play-dgraph-io.nginx.conf:2]: I'm not sure, but that change was made manually by Daniel.

**Comment by MichelDiz** [comment]
We don't need to hide this specific token. But if you wanna add this, let's see.

There's no auto-deployment for now. The instance is manually configured. I see no sign of scripts in the EC2 instance.

**Pull request merged**